shell/common/animator.h

```diff
@@ -108,7 +108,7 @@ class Animator final {
   bool frame_scheduled_;
   int notify_idle_task_id_;
   bool dimension_change_pending_;
-  SkISize last_layer_tree_size_;
+  SkISize last_layer_tree_size_ = { 0, 0 };
```

Comment:
The format checker script wants different formatting - see https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket.appspot.com/8862735462431704704/+/steps/format_and_dart_test/0/stdout

Reply:
Done.